### PR TITLE
feat: disable refresh on non dev build

### PR DESCRIPTION
**Pull request by MCozhusheck**
Description
---
fixes: #502 
Disabled whole context menu (menu on right click) in non dev build.

Motivation and Context
---
Disable refresh action for normal user which in current state introduces bugs to the application.

How Has This Been Tested?
---
Launch app in dev and release mode and right click to check if context menu appears.

What process can a PR reviewer use to test or verify this change?
---

Breaking Changes
---

- [x] None
- [ ] Requires data directory on base node to be deleted
- [ ] Requires hard fork
- [ ] Other - Please specify

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -18,8 +18,25 @@ import CriticalErrorDialog from './containers/CriticalErrorDialog/CriticalErrorD
...
+    });
+
+    document.addEventListener('contextmenu', function (event) {
+        event.preventDefault();
+    });
+};
+
 export default function App() {
     useLangaugeResolver();
+    disableRefresh();
```
Comment (mmrrnn) [inline src/App.tsx:39]: Use hook which returns fn cleaning at every rerender(removes this event listener)

Comment (MCozhusheck) [reply src/App.tsx]: done

**Review by brianp — APPROVED** [review]
Tested and on prod builds the menu is disabled.

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.5.60 and when I right click within the release build, I don't see a menu

**Pull request merged**